emulator/iSkipper/examples/capture/capture.ino

```diff
@@ -29,23 +30,29 @@ void setup()
 void loop()
 {
   char tmp[50];
-  iClickerPacket r;
+  iClickerPacket r;  
 
   //see if there is a pending packet, check if its an answer packet
-
-  while (recvBuf.pull(&r)) {
+  while (recvBuf.pull(&r) && r.type == PACKET_ANSWER) {
     uint8_t *id = r.packet.answerPacket.id;
     char answer = iClickerEmulator::answerChar(r.packet.answerPacket.answer);
     snprintf(tmp, sizeof(tmp), "Captured: %c (%02X, %02X, %02X, %02X) \n", answer, id[0], id[1], id[2], id[3]);
     Serial.println(tmp);
   }
 
   delay(100);
-
 }
 
 
 void recvPacketHandler(iClickerPacket *recvd)
 {
+    if (SEND_ACKS && recvd->type == PACKET_ANSWER) {
+      Serial.println("Sending ACK");
```

Comment:
It is undefined behavior in the Arduino ecosystem to print in an ISR. I think we can just delete this line.